src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -500,6 +488,78 @@ def _object_href(object_id: str, path: str = "", requested_pack: str = "") -> st
+    Prefers the H1 line from the markdown body, then the YAML
+    ``title:`` frontmatter, then the path basename.  This is
+    what BL-087's entry button passes through to BL-086's
+    composer as the session's anchor title.
+    """
+    for line in markdown.splitlines():
+        if line.startswith("# "):
+            text = line[2:].strip()
+            if text:
+                return text
+            break
+        if line.startswith("## ") or line.startswith("### "):
```

Ruff (0.15.12)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)